### ISSUE TITLE
Debug flag issue

### ISSUE THREAD

**Issue by cabaneros** [baemingo/serverless-tscpaths#5]
Hello and thank you for this plugin! I've been using it for a while. There is an issue tho, if I don't run serverless with `SLS_DEBUG=*`, I get `unknown flag --silent` error. My fix suggestion is
```javascript
 private async hook() {
    this.serverless.cli.log('Fixing paths');
    const flag = this.isDebug && '--verbose';
```

I could open a PR for it. Thank you!

**Comment by tgrades** [comment]
Hi. The better solution would be to customize flags depending on the lib, but maybe it's too complicated, so lets keep it simple and remove the `--silent` flag as you suggested.
So yeah, if you can create a PR, it would be great!

**Comment by tgrades** [comment]
Merged your pr https://github.com/baemingo/serverless-tscpaths/pull/6 and published new `0.0.6` version
